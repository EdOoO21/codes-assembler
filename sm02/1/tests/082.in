575 3 34 -402 984 688 662 -633 618 -659 392 -467 -680 562 870 -932 -113 -35
