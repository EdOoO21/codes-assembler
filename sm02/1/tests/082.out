-35
-113
-932
870
562
-680
-467
392
-659
618
-633
662
688
984
-402
34
3
575
